README.md

```diff
@@ -0,0 +1,705 @@
+
+#### Step: Render the definition
+
+The [chuhlomin/render-template@v1](https://github.com/marketplace/actions/render-template) generates from the `docker-library-definition.tmpl` using data (`env.tags`, `env.last_commit`, `matrix.version_major` and `matrix.image_types`) file `official-images/library/almalinux.version_major.image_types`. The file is a part of Docker Library *Definition File*.
+
+#### Step: Upload the definition for *version_major* *image_types*
+
+The step uses [actions/upload-artifact@v4](https://github.com/actions/upload-artifact) to store as artifact generated on previous step `official-images/library/almalinux.version_major.image_types`. The artifact is named against `version_major` and `image_type`, like `definition-version_major.image_types`.
+
+Artifacts are used to transfer files between different jobs of the same workflow. The artifact is zip archive of the file without file-path included.
+
+That is also possible to download artifacts via GitHub Action's web interface.
+
+### Job: Create Pull Request with the new definition file
+
+> The job is skipped if the *'Publish to Docker Official Library'* isn't checked (`inputs.pr` set into `false`)
+
+#### Step: Checkout *official-images*, branch 'master'
+
+The [actions/checkout@v4](https://github.com/actions/checkout/) checkouts *container-images* into branch 'master'. The repository directory is located at `/home/runner/work/container-images/official-images`.
+
+That's your fork of [docker-library/official-images](https://github.com/docker-library/official-images) repository.
+
+#### Step: Sync *official-images* with upstream
+
+The step is written in bash. It adds the [official-images upstream](https://github.com/docker-library/official-images) repository, set local 'master' branch to track corresponded from the upstream, and tries to rebase with the 'upstream/master'.
+
+#### Step: Download all definitions
+
+Uses [actions/download-artifact@v4](https://github.com/actions/download-artifact) to download multiple (`merge-multiple: true`) artifacts with generated definitions. The files are saved into the `official-images/library/` directory.
+
+#### Step: Create head of *official-images/library/almalinux*
+
+Creates heading for the Docker *Definition File*.
+
+#### Step: Merge definitions into *official-images/library/almalinux*
+
+The step is written in bash. It appends the `official-images/library/almalinux` *Definition File* with downloaded on the previous step definitions `official-images/library/almalinux.version_major.image_types`.
+
+#### Step: Prepare date stamp
+
+Generates `date_stamp` in format *YYYYMMDD*. It is used in commit message and pull request title.
```

Comment:
Generates `date_stamp` in the *YYYYMMDD* format. It is used in the commit message and pull request title.

Reply:
Done.